.github/workflows/release.yml

```
@@ -0,0 +1,61 @@
+    tags:
+      - '*'
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    # Only run for valid semver tags without 'v' prefix
+    if: |
+      contains(github.ref, 'refs/tags/') && 
```

actionlint (1.7.7)

undefined function "matches". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)